lib/async/http/endpoint.rb

```diff
@@ -18,16 +18,26 @@ module Async
 	module HTTP
 		# Represents a way to connect to a remote HTTP server.
 		class Endpoint < ::IO::Endpoint::Generic
+			SCHEMES = ['HTTP', 'HTTPS', 'WS', 'WSS'].to_h do |scheme|
+				[scheme.downcase, URI.scheme_list[scheme]]
+			end
```

Comment:
I would just write out the Hash as a literal. Less code that needs to execute at load time.

Reply:
Yep, that makes sense.